### PR TITLE
Separate error messages on Installation test

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Seperate error messages on Installation test
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | Install test should be good, if failed, the right message should be displayed
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23946)

### CODE REVIEW

**Review by nesrineabdmouleh — DISMISSED** [review]

File: tests/UI/campaigns/sanity/01_installShop/01_installShop.js [hunk 1]
```diff
@@ -99,10 +99,116 @@ describe('Install Prestashop', async () => {
...
+        },
+    },
+
+  ];
+
+  tests.forEach((test, index) => {
+    it(`should installation step '${test.args.step.name}' be finished`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `CheckStep${index}`, baseContext);
+
+      const stepFinished = await installPage.IsInstallationStepFinished(
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/01_installShop/01_installShop.js]:
```suggestion
      const stepFinished = await installPage.isInstallationStepFinished(
```

File: tests/UI/pages/install/index.js [hunk 2]
```diff
@@ -178,27 +178,76 @@ class Install extends CommonPage {
...
+  }
+
+  /**
+   * Check if step installation is finished
+   * @param page
+   * @param step
+   * @param timeout
+   * @returns {Promise<boolean>}
+   */
+  async IsInstallationStepFinished(page, step, timeout = 30000) {
```
Comment (Progi1984) [inline tests/UI/pages/install/index.js]:
```suggestion
  async isInstallationStepFinished(page, step, timeout = 30000) {
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**